examples/timers/src/main.py

```diff
@@ -0,0 +1,107 @@
+from kybra import Duration, ic, nat8, query, Record, TimerId, update
+
+
+class StatusReport(Record):
+    single: bool
+    inline1: nat8
+    inline2: nat8
+    inner: str
+    repeat: nat8
+
+
+class TimerIds(Record):
+    single: TimerId
+    inline1: TimerId
+    inline2: TimerId
+    inner: TimerId
+    repeat: TimerId
+
+
+status: StatusReport = {
+    "single": False,
+    "inline1": 0,
+    "inline2": 0,
+    "inner": "",
+    "repeat": 0,
+}
+
+
+@update
+def clear_timer(timer_id: TimerId):
+    ic.clear_timer(timer_id)
+    ic.print(f"timer {timer_id} cancelled")
+
+
+@update
+def set_timers(delay: Duration, interval: Duration) -> TimerIds:
+    captured_value = "🚩"
+
+    def inner_closure():
```

Comment:
Let's discuss this, the examples should be identical, and I'm not sure why we have the inner function here